string room_name
---
bool accepted
string message
